CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Breaking changes:
 
+- [#1645](https://github.com/rails-api/active_model_serializers/pull/1645) Expanded KeyTransform into Transform. Changed :dashed transform to :dash. (@remear)
```

Comment:
Sorry, I see now what you were actually asking. The question totally went over my head last night. `KeyTransform` -> `Transform` because it no longer concerns transforming only keys.